tpm2/keydata.go

```diff
@@ -238,31 +248,74 @@ func (k *SealedKeyObject) PCRPolicyCounterHandle() tpm2.Handle {
 
 // WriteAtomic will serialize this SealedKeyObject to the supplied writer.
```

Comment:
should the doc comment mention something about this not being usable with new objects? is this going away at some point?

Reply:
This is fixed by PR #250 now.